en/integrations/event-management/sc-bsm.md

```diff
@@ -3,124 +3,255 @@ id: sc-hp-bsm
+### Standard parameters
 
-**Remarks**
+All stream connectors can use a set of optional parameters that are made available through Centreon stream connectors lua modules.
 
-* The default value of 2 for `log_level` is fine for initial troubleshooting, but can generate a huge amount of logs if you monitor a lot of hosts. In order to get less log messages, you should tune this parameter.
-* The default value of 1 for `max_buffer_size` works fine and ensures the best response times. You might want to tune it (*ie.* increase it) if you have an important amount of data to send to BSM.
+All those parameters are documented **[here](https://github.com/centreon/centreon-stream-connector-scripts/blob/master/modules/docs/sc_param.md#default-parameters)**.
 
----------------
+Some of them are overridden by this stream connector.
 
-## How to Uninstall
+| Type   | Name                | Default value for the stream connector |
+| ------ | ------------------- | -------------------------------------- |
+| string | accepted_categories | neb                                    |
+| string | accepted_elements   | host_status,service_status             |
 
-1. Login to the Centreon WUI using an administrator account.
-2. Navigate to the **Configuration** > **Pollers** menu and select **Broker configuration**.
-3. Click on the **central-broker-master** broker configuration object and navigate to the **Output** tab.
-4. Delete the **Generic - Stream connector** output by clicking on the red circled cross at the end of the line.
-5. Save your configuration, then navigate to the **Configuration** > **Pollers** menu and select **Pollers**.
-6. Select the **Central** poller and click on **Export configuration**.
-7. Keep **Generate Configuration Files** and **Run monitoring engine debug (-v)** checked and select **Move Export Files** and then click on the **Export** button.
-8. Restart the `cbd` service:
+## Event bulking
 
-```bash
-systemctl restart cbd
+This stream connector is not compatible with event bulking. Meaning that the option `max_buffer_size` can't be higher than 1
+
+## Event format
+
+This stream connector will send event with the following format.
+
+### status event
+
+```xml
+{
+ "<event_data>"
+        "<hostname>"  hostname  "</hostname>"
+        "<svc_desc>"  service_description  "</svc_desc>"
+        "<state>" self.sc_event.event.state  "</state>"
+        "<last_update>" self.sc_event.event.last_update  "</last_update>"
+        "<output>"  string.match(e.output, "^(.*)\n")  "</output>"
+        xml_service_severity
+        "<url>"  xml_url  "</url>"
+        "<source_host_id>"  ifnil_or_empty(self.sc_event.event.host_id, '0')  "</source_host_id>"
+        "<source_svc_id>"  ifnil_or_empty(self.sc_event.event.service_id, '0')  "</source_svc_id>"
+        "<scheduled_downtime_depth>"  ifnil_or_empty(self.sc_event.event.scheduled_downtime_depth, '0')  "</scheduled_downtime_depth>"
+ "</event_data>"
+}
```

Comment:
this is a copy paste from the code, this is not the xml that is sent 😈 

this stream connector handles services and hosts, you need to provide the example for both kind of events